src/pages/Internship.jsx

```diff
@@ -1,180 +1,152 @@
+                    <br/>
+                    Why Intern for FUSION
                 </div>
-            </section>
-
-            <section className="bg-white w-full h-[761.5px] flex items-center justify-center"> {/* random pics?*/}
-                <div className="carousel rounded-box mt-[155px] gap-[64px]">
-                    <div className="carousel-item">
-                        <img className="w-[613.5px] h-[372px] object-cover" src={fam_placeholder}/>
-                    </div>
-                    <div className="carousel-item">
-                        <img className="w-[613.5px] h-[372px] object-cover" src={fam_placeholder}/>
+                <Testimonial
+                    Name="Cayz Paclibar"
+                    Quote="“The FUSION internship was my stepping stone to becoming a better leader,
+                     teaching me vital skills in public speaking, communication, and organization.
+                      FUSION has been the heart of my college experience because of everything it’s taught me <3”"
+                    Position="FU Class >> Secretary ‘24-’25"
+                    Image= {CayzPhoto}
+                    Color= "#F4F4F4"
+                    Mirrored={true}
+                />
+                <Testimonial
+                    Name="Jhaneen Uy"
+                    Quote="“Participating in FUSION's internship broadened my perspective of the organization,
+                     pushing me beyond my comfort zone to engage more confidently in larger social environments and develop my stage presence. “"
+                    Position="SI Class >> Internal Vice President ‘24-’25"
+                    Image= {JhaneenPhoto}
+                    Color= "#F4F4F4"
+                    Mirrored={false}
+                />
+                <Testimonial
+                    Name="Arlaghn Cayanan"
+                    Quote="“Becoming an intern gave me hands-on experience with FUSION's programs, allowing me to grow both professionally 
+                    and personally in this space. My intern experience inspired me to pursue a bigger role, ultimately leading to my application for a board position.”"
+                    Position="ON Class >> Athletics Co-Director ‘24-’25"
+                    Image= {ArlaghnPhoto}
+                    Color= "#F4F4F4"
+                    Mirrored={true}
+                />
+                <img src={Dots2} className="absolute top-0 left-0 opacity-50"/>
+                <img src={WhyInternBG} className="absolute bottom-0 right-0 -z-10"/>
+                <img src={WhyInternRectangle1} className="absolute top-32 left-32"/>
+                <img src={WhyInternRectangle2} className="absolute bottom-64 left-48"/>
+                <img src={WhyInternRectangle3} className="absolute left-32"/>
+                <img src={WhyInternRectangle2} className="absolute top-64 right-48"/>
+                <img src={WhyInternRectangle3} className="absolute top-32 right-32"/>
+            </div>
```

Comment:
Note that when it comes to these colors, even though it may look good on your end and you specify the colors, it might not in another persons screen, for example mine is dark theme and I don't see a white background so just ensure that you specify the colors